certification/internal/engine/podman.go

```diff
@@ -177,3 +148,127 @@ func (p *podmanEngine) WaitContainer(containerId string, waitOptions cli.WaitOpt
 	log.Info("container reached a running state")
 	return true, nil
 }
+
+func (p *podmanEngine) RunSystemContainer(containerName string) (*cli.PodmanOutput, error) {
+	log.Debug(fmt.Sprintf("Generating Systemd file for container %s", containerName))
+	cmdArgs := []string{"generate", "systemd", "--new", "--files", "--name", containerName}
+
+	output, err := RunCommand(false, "podman", cmdArgs)
```

Comment:
You have a "magic boolean" here. One has to go and read the RunCommand function to see what that boolean means. It wouldn't be a bad idea to make a 'var privileged bool' at the top of this function, use it throughout, and just change it before each call.

Reply:
Sounds like a great idea! Will update the PR shortly :)